src/components/CloudIcon/index.jsx

```diff
@@ -2,16 +2,18 @@ import React from 'react'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import './styles.css'
 
-const CloudIcon = props => {
+import CloudIcon from "cozy-ui/transpiled/react/Icons/Cloud";
```

Comment:
weird to have " ";

Reply:
I squashed the next lint commit into this one, this way we do not have the ";" (caused by the codemod).